examples/sglang_bench/bench_sharegpt.py

```
@@ -434,6 +692,28 @@ def main():
     finally:
         log("Shutting down server...")
         kill_process_tree(process.pid)
+        for p in peer_processes:
+            try:
+                # Kill the remote sglang worker via SSH, then close the local SSH process.
+                peer_host = p.args[5]  # ssh cmd: ["ssh", "-o", OPT, "-o", OPT, HOST, remote_cmd]
+                subprocess.run(
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)